### PR TITLE
fixed error in ip address

### LINKED ISSUE #63: Test extensively
Test extensively all the possible use cases and scenarios to make sure that everything works as expected and there are no problems or bugs. Report (create issues) and fix any problems/bugs.

Create a testing document that describes all the possible cases and scenarios that need to be tested, what values of `settings.sh` should be used, what scripts or commands to run, etc.

This is important because extensive testing needs to be done whenever new features are added or modifications are done to the scripts, to make sure that everything works correctly.

This will also make easy for other people to perform testing correctly, without missing any steps (related also to https://github.com/docker-scripts/dev--LTSP/issues/18).

### PR DESCRIPTION
**Pull request by d78ui98**
Default ip set by `ltsp-config dnsmasq` is different for linux mint `tara`
closes #63

### CODE REVIEW

**Review by dashohoxha — COMMENTED** [review]
@akash0x53 please review them because I am not able to test them (and I will also be a bit busy on the following days).

File: install.sh [hunk 1]
```diff
@@ -62,7 +62,7 @@ else
     echo "There is an existing DHCP server running"
     echo "LTSP server won't provide DHCP services.."
     sed -i /etc/dnsmasq.d/ltsp-server-dnsmasq.conf \
-        -e "/192.168.1.0,proxy\$/ c dhcp-range=${NETWORK}.0,proxy"
+        -e "/192.168.111.0,proxy\$/ c dhcp-range=${NETWORK}.0,proxy"
```
Comment (akash0x53) [inline install.sh:66]: Why `111` and not `222`? Is this fixed?

File: install.sh [hunk 2]
```diff
@@ -6,7 +6,7 @@ NETWORK="$(echo $LAN_IP | cut -d'.' -f1-3)"
 
 # Updating packages
 apt update --yes
-apt upgrade --yes
+DEBIAN_FRONTEND=noninteractive apt upgrade --yes
```
Comment (akash0x53) [inline install.sh]: It would be helpful to mention the reason why this variable is set.

File: install.sh (hunk 1 above)
Comment (d78ui98) [reply install.sh:66]: Yes. 
`ltsp-config dnsmasq` generates the fixed ip address

**Review by dashohoxha — APPROVED** [review]
Let's get going.

**Pull request merged**